sample_trial/definition.json

```diff
@@ -34,7 +34,7 @@
             "image_name": "ghcr.io/ndph-arts/config-server",
             "image_tag": "main-latest",
             "git_uri": "https://github.com/NDPH-ARTS/mts-trial-deployment-config.git",
-            "search_paths": "services/practitioner-service"
+            "search_paths": "sample_trial/services, sample_trial/services/practitioner-service"
```

Comment:
Isn't sample_trial/services enough to include all subfolders?

Reply:
Its not enough. verified.

Reply:
is the wildcard * not picked up? According to spring documentation (for app properties) it should work with * - though not sure when reading from json...

Reply:
I can improve by:
sample_trial/services, sample_trial/services/*

this will catch all the different future services.
however, this: sample_trial/services* , does not work...